### ISSUE TITLE
Watercooler - Q3 2018

### ISSUE THREAD

**Issue by Changaco** [liberapay/salon#244]
This is Liberapay's quarterly "what's up?" thread. Previous: #218.

**Comment by Changaco** [comment]
[Alternatives To Patreon For Porn Creators](http://msnaughty.com/blog/2018/07/03/alternatives-to-patreon-for-porn-creators/) (something to link to when telling people that we can't help adult content creators).

**Comment by Changaco** [comment]
[Liberapay & MANGOPAY](https://www.mangopay.com/blog/liberapay/)

**Comment by jorgesumle** [comment]
*Probably off-topic*: comment to "Liberapay & MANGOPAY"

> the platform is not in line with its terms of use

They are not very specific.

> MANGOPAY remains at full disposal to Liberapay for any clarification needed.

Doublespeak. We don't even know for sure when they'll cut us off, or when they can *legally* cut us off. They should have cited the exact term Liberapay is infringing from their contract, they haven't done that. Maybe someone paid them to boycott Liberapay, who knows? With so much transparency one only can speculate.

Regardless, I think that the response to the crisis has been great, and I hope this will help Liberapay be more resilient.

**Comment by mattbk** [comment]
Wow, guess the publicity is getting under their skin.

**Comment by Changaco** [comment]
@mattbk For the record, we warned them that a hasty contract termination would damage their reputation.

@jorgesumle It's difficult to be off-topic in the Watercooler thread. :slightly_smiling_face:

**Comment by jorgesumle** [comment]
[Liberapay is in trouble](https://www.peppercarrot.com/en/article444/liberapay-is-in-trouble)

**Comment by jorgesumle** [comment]
[Hyperbola suspends its donation account in Liberapay](https://www.hyperbola.info/news/hyperbola-suspends-its-donation-account-in-liberapay/)

**Comment by FrostKnight** [comment]
I may be wrong jorgesumle, but I believe Hyperbola closed their account because of this: 
https://www.privacytools.io/#ukusa 
I would ask them for clarification, if you want I can though.
I am a user of this distro, that is why I am offering to help if I can.

**Comment by MartinDelille** [comment]
We are now on Telegram through:
- A public channel broadcasting information about the plateform: https://t.me/liberapay
- A group to chat between user and developer: https://t.me/joinchat/HZKyvg8F5-ncY5c1ukzDbA

**Comment by trebmuh** [comment]
https://medium.com/le-blog-qwant/5-raisons-de-passer-%C3%A0-brave-qwant-68df3566de5a

**Comment by MartinDelille** [comment]
I think the liberapay codebase respect this 5 principles: https://medium.com/@george.seif94/these-5-clean-code-tips-will-dramatically-improve-your-productivity-b20c152783b

### TL;DR

1. If it isn’t tested, it’s broken
1. Choose meaningful names
1. Classes and functions should be small and obey the Single Responsibility Principle (SRP)
1. Catch and handle exceptions, even if you don’t think you need to
1. Logs, logs, logs

**Comment by mattbk** [comment]
https://twitter.com/MutantStd/status/1041460158537375744

> I’m closing the Liberapay account.
>Hey, just to let you know, I’m going to close the Liberapay account… as soon as Liberapay lets me.

https://blog.mutant.tech/index.php/2018/09/16/im-closing-the-liberapay-account/

This is the sort of thing that happens when communication with users is poor.

**Comment by MartinDelille** [comment]
I wrote a first answer to try opening the dialog.
